hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -2205,6 +2272,167 @@ private void listStatusFindKeyInTableCache(
     return fileStatusList;
   }
 
+  public List<OzoneFileStatus> listStatusV1(OmKeyArgs args, boolean recursive,
+      String startKey, long numEntries, String clientAddress)
+          throws IOException {
+    Preconditions.checkNotNull(args, "Key args can not be null");
+
+    List<OzoneFileStatus> fileStatusList = new ArrayList<>();
+    if (numEntries <= 0) {
+      return fileStatusList;
+    }
+
+    String volumeName = args.getVolumeName();
+    String bucketName = args.getBucketName();
+    String keyName = args.getKeyName();
```

Comment:
Additionally, startKey needs to be normalized when FileSystemPaths enabled, related JIRA: HDDS-4117

Reply:
Thanks a lot for pointing out this. Since the startKey value is internally taken from the OMKeyInfo.keyName, I thought the path would have already been in the normalized form. I saw startKey normalization in PR #1451, I would like to hear from @bharatviswa504 about the plans and follow the same logic here as well.

Reply:
Seems the keyName in OmKeyArgs already be normalized. But startKey is not included in that, it's an independent parameter passed in here. So the PR #1451 did the fix for this.

Reply:
Noted and will add TODO in V1 code. I will watch #1451 progress and will incorporate that logic in V1 once that PR is merged. Hope that make sense to you?

Reply:
Makes sense to me.